surveySimPP/modules/PPCalculateApparentMagnitude.py

```diff
@@ -18,30 +19,18 @@ def PPCalculateApparentMagnitude(observations, phasefunction, mainfilter, otherc
            mainfilter     : string
            othercolours   : array of strings
            resfilters     : array of strings
-           filterpointing : string
    
     Output: observations: amended pandas DataFrame
  
-    Usage: observations=PPCalculateApparentMagnitude(observations, phasefunction, mainfilter, othercolours, resfilters, filterpointing)
+    Usage: observations=PPCalculateApparentMagnitude(observations, phasefunction, mainfilter, othercolours, resfilters)
     """
     
     pplogger = logging.getLogger(__name__)
  
     pplogger.info('Calculating apparent magnitudes...')
     observations=PPCalculateApparentMagnitudeInFilter.PPCalculateApparentMagnitudeInFilter(observations, phasefunction, mainfilter)        
     
-
-    
-    pplogger.info('Hooking colour and brightness information...')
-    i=0
-    while (i<len(othercolours)):
-         observations=PPhookBrightnessWithColour.PPhookBrightnessWithColour(observations, mainfilter, othercolours[i], resfilters[i+1])         
-         i=i+1
- 
-    observations=observations.reset_index(drop=True)
-
-    
-    pplogger.info('Resolving the apparent brightness in a given optical filter corresponding to the pointing...')
-    observations=PPMatchPointingsAndColours.PPMatchPointingsAndColours(observations,filterpointing)
+    pplogger.info('Selecting and applying correct colour offset...')
+    observations=PPResolveMagnitudeInFilter.PPResolveMagnitudeInFilter(observations,mainfilter,othercolours,resfilters)
```

Comment:
This function needs a better name. 

How in reading the names of the functions would someone know the difference between PPCalculateApparentMagnitudeInFilter and PPResolveMagnitudeInFilter? They sound like exactly the same thing and they are not.